sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/ScriptTransformation.scala

```diff
@@ -58,11 +58,12 @@ case class ScriptTransformation(
     child.execute().mapPartitions { iter =>
       val cmd = List("/bin/bash", "-c", script)
       val builder = new ProcessBuilder(cmd)
+      builder.redirectErrorStream(true)
```

Comment:
If your goal is to just consume the error stream, you don't need to redirect to stdout. In fact, are you sure it's safe? this may not be expected?